cmd/rp-adapter-vue/package.json

```diff
@@ -13,7 +13,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.14.0",
     "@fortawesome/free-solid-svg-icons": "^5.14.0",
     "@fortawesome/vue-fontawesome": "^0.1.10",
-    "@trustbloc/wallet-js-client": "0.1.6",
+    "@trustbloc-cicd/wallet-adapter-web": "0.1.7-snapshot-0bd7297",
```

Comment:
why just reference wallet-adapter-web directly

Reply:
it's in same project

Reply:
no need for snapshot

Reply:
use ../

Reply:
I noticed when we do npm install local it installs all contents outside webpack dist, so it is gets heavy.